### PR TITLE
FOGL-132 PR tester script added

### PR DESCRIPTION
**Pull request by ashish-ScaleDB**
[FOGL-132](https://scaledb.atlassian.net/browse/FOGL-132) - Add Pr tester script

### CODE REVIEW

File: pr_tester.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+# DB setup
+jenkins_db_user='jenkins'
+foglamp_db_user='foglamp'
+postgres_db_user='postgres'
+
+PGPASSWORD=${postgres_db_user} psql -U ${postgres_db_user} -h localhost -f src/sql/foglamp_ddl.sql ${postgres_db_user}
+PGPASSWORD=${foglamp_db_user} psql -U ${foglamp_db_user} -h localhost -f src/sql/foglamp_init_data.sql ${foglamp_db_user}
+PGPASSWORD=${foglamp_db_user} psql -c "GRANT ALL ON DATABASE ${foglamp_db_user} to ${jenkins_db_user};" -U ${foglamp_db_user} -h localhost
```
Comment (ashwinscale) [inline pr_tester.sh]: why is the jenkins users of concern here?

**Comment by ashwinscale** [comment]
The pr_tester_job link is out of date.

File: pr_tester.sh (hunk 1 above)
Comment (ashish-ScaleDB) [reply pr_tester.sh]: This was needed before merge https://scaledb.atlassian.net/browse/FOGL-506 into develop. These changes are stale now as per latest develop, will be fix in another round commit.

**Comment by ashish-ScaleDB** [comment]
@ashwinscale @praveen-garg 
Ready for review!!

Jobs can be found here:
1) [lint check job](http://jenkins.scaledb.com:8080/job/foglamp_lint_check_tester/)
2) [unit test and integration](http://jenkins.scaledb.com:8080/job/foglamp_unit_and_integration_tester/)

NOTE: 1) TODO: Add github usernames/organization to whitelist on PR tester jobs. (pull request, push events from these users will trigger the PR tester)

**Comment by ashwinscale** [comment]
Must they be two separate jobs?

**Comment by ashwinscale** [comment]
Can we simplify the name of the job for readability. 

For example:
<a title="pr tester script added" href="https://github.com/foglamp/FogLAMP/pull/180">PR #180</a>: pr tester script added

To:
PR-180 or better yet FOGL-XX

**Comment by ashish-ScaleDB** [comment]
Github Pull Request Builder plugin is not compatible with the Pipeline one, so it is not possible to use automatically trigger a pipeline when a new pull request is opened  or any GitHub push.
Here is the [issue](https://issues.jenkins-ci.org/browse/JENKINS-26591) to track the problem. So I have used these 2 jobs separately as a free template jobs for now.

File: pr_tester.sh [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        source build.sh -c
+
+        # activate virtual env
+        source build.sh -a
+
+        # install develop dependencies
+        make develop
+
+        # pylint check and remove src/python from path
+        OUTPUT="$(echo $LINT_CHECK_FILES | sed 's/src\/python\///g')"
```
Comment (vaibhav-ScaleDB) [inline pr_tester.sh]: Rename OUTPUT to something better, these are the name of the files without the absolute path

**Pull request merged**